src/App.js

```diff
@@ -13,7 +13,7 @@ function App(){
     } 
     useEffect(() => {
       fetchInfo();
-    }, []);
```

Comment:
This seems like an incorrect change. Omitting the empty array for the dependencies argument will cause the effect to re-run every render, meaning data will be fetched every re-render instead of just at the start.

Reply:
Okay, I understand.